<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogicAnalyzer</class>
 <widget class="QMainWindow" name="LogicAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logic Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="CoordinatePlotWidget" name="graphicsLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="actionSingle"/>
  </widget>
  <action name="actionExpand">
   <property name="text">
    <string>Expand</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-start-7.png</normaloff>:/openicon/icons/media-playback-start-7.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Experiment.qrc">
     <normaloff>:/openicon/icons/media-playback-stop-7.png</normaloff>:/openicon/icons/media-playback-stop-7.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSingle">
   <property name="text">
    <string>Single</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CoordinatePlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>CoordinatePlotWidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
